transforms/csdl/preprocess_csdl.xsl

```diff
@@ -926,6 +965,18 @@
           </xsl:call-template>
        </xsl:copy>
     </xsl:template>
+
+    <!-- If the parent "Annotations" tag already exists modify it -->
+    <!-- Remove readability only for teams entity set -->
+    <xsl:template match="edm:Schema[@Namespace='microsoft.graph']/edm:Annotations[@Target='microsoft.graph.GraphService/teams']">
+      <xsl:copy>
+        <xsl:copy-of select="@*|node()"/>
+           <xsl:call-template name="ReadRestrictionsTemplate">
+              <xsl:with-param name="readable">false</xsl:with-param>
```

Comment:
We should isolate `readable` of `false` to just v1.0 metadata. `GET /teams` is [supported in `beta`](https://learn.microsoft.com/en-us/graph/api/teams-list?view=graph-rest-beta&tabs=http).

Reply:
Anyone know how we can be able to target the file path to the source XML? We might need to target the file path name to be able make this conditional decision. Trying [document-uri()](https://learning.oreilly.com/library/view/xslt-2nd-edition/9780596527211/re83.html) fails with the error: _document-uri()' is an unknown XSLT function._
@peombwa @baywet

Reply:
we could imagine adding another parameter and updating the scripts if that makes things easier.

Reply:
+1 on using a parameter instead. This will make it easier to apply API version specific annotations in the future.

Reply:
All agreed. Will transfer this to be parameter-based

Reply:
This has been resolved